tests/integration/api/usage_test.go

```diff
@@ -0,0 +1,35 @@
+package api
+
+import (
+	"testing"
```

Comment:
This test is really non-intuitive. If I was new to the codebase I might just delete it, not undertanding its purpose because it's very side-affecty.